### PR TITLE
`Signing`: updated to use production public key

### PR DESCRIPTION
**Pull request by NachoSoto**
Depends on #2272 and #2278.
Depends on https://github.com/RevenueCat/khepri/pull/5232.

### CODE REVIEW

**Comment by NachoSoto** [comment]
These tests should pass now 🎉

File: Tests/UnitTests/Misc/SigningTests.swift [hunk 1]
```diff
@@ -110,6 +110,37 @@ class SigningTests: TestCase {
...
+        -H 'Connection: close' \
+        -H 'Content-Length: 54' \
+        -d '{"app_user_id": "test", "new_app_user_id": "new_user"}'
+         */
+
+        // swiftlint:disable line_length
+        let response = """
+        {"request_date":"2023-02-10T18:04:21Z","request_date_ms":1676052261449,"subscriber":{"entitlements":{},"first_seen":"2023-02-07T18:26:02Z","last_seen":"2023-02-07T18:26:02Z","management_url":null,"non_subscriptions":{},"original_app_user_id":"new_user","original_application_version":null,"original_purchase_date":null,"other_purchases":{},"subscriptions":{}}}\n
+        """
+        let expectedSignature = "XOhM+HIJRMiAyJ24wlbqq/F1b20BzvAraVxYqknhF4WNv2qmbEBEEq1Lv7WAxXWoZs6do1cIdrWlPOpZ/pMmaJ3NN07AOMP3/9MzYhT1TAc="
```
Comment (NachoSoto) [inline Tests/UnitTests/Misc/SigningTests.swift:131]: We need to update this once the new key is deployed for this test to pass.

**Comment by NachoSoto** [comment]
This is ready now.

**Review by aboedo — APPROVED** [review]

**Comment by NachoSoto** [comment]
Loading keys no longer throws /cc @aboedo

**Pull request merged**